src/components/App/App.jsx

```diff
@@ -9,7 +9,7 @@ import Sidebar from "../NavBar/Sidebar";
 import Notification from "../Notification/Notification";
 import Schedule from "../Schedule/Schedule";
 import Sponsors from "../Sponsors/Sponsors";
-
+import "./App.css";
```

Comment:
There should probably be a new line after this and before the function declaration